gthockey-iOS/gthockey-iOS/CustomUIComponents/PillButton.swift

```diff
@@ -57,4 +58,36 @@ class PillButton: UIButton {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // MARK: Loading Animation
+
+    private var originalButtonText: String?
+
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let activityIndicator = UIActivityIndicatorView()
+        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
+        activityIndicator.color = .black
```

Comment:
I think the default color for the text is white. Have you seen what it looks like when the activity indicator is white? I think it would be a cleaner transition to make the colors the same.